src/Database.ts

```diff
@@ -358,13 +355,6 @@ export class Database<
     })
   }
 
-  /**
-   * Performs a database migration.
-   */
-  async migrate (config?: MigrationParams) {
```

Comment:
Keep, but we'll create a new instance of your migration class here, like

```
const migrate = new Migration(<params >)

// do migration

Reply:
I'll do it in another PR